File: app/src/main/java/ds/photosight/ui/CategoriesFragment.java
package ds.photosight.ui;

import ds.photosight.Constants;
import ds.photosight.R;


public class CategoriesFragment extends ListFragmentAbstract implements Constants {

	@Override
	protected ListAdapter initList() {
		return new ListAdapter(getActivity(), R.layout.simple_list_item_activated_1, getResources().getStringArray(
				R.array.categories_array));

	}


	@Override
	protected int getListType() {
		return TAB_CATEGORIES;
	}

}

File: app/src/main/java/ds/photosight/ui/CategoriesFragment.kt
package ds.photosight.ui

import ds.photosight.Constants
import ds.photosight.R


class CategoriesFragment : ListFragmentAbstract(), Constants {

    override fun initList(): ListAdapter? =
            ListAdapter(activity!!, R.layout.simple_list_item_activated_1, resources.getStringArray(R.array.categories_array))


    override val listType: Int get() = Constants.TAB_CATEGORIES

}
